application/views/admin/globalsettings/_security.php

```diff
@@ -147,23 +147,29 @@
                         <?php eT("IP whitelist:"); ?>
                     </label>
                     <textarea class="form-control" id='loginIpWhitelist' name='loginIpWhitelist'><?php echo htmlspecialchars((string) Yii::app()->getConfig('loginIpWhitelist')); ?></textarea>
-                    <span class='hint'><?php eT("List of IP addresses to exclude from the maximum login attempts check. Separate each IP address with a comma or a new line."); ?></span>
+                    <div class='form-text'><?php eT("List of IP addresses to exclude from the maximum login attempts check. Separate each IP address with a comma or a new line."); ?></div>
                 </div>
 
                 <div class="mb-3">
                     <label class="form-label" for='maxLoginAttempt'>
                         <?php eT("Maximum number of attempts:"); ?>
                     </label>
                     <div class="">
-                        <input class="form-control" type="number" min="0" name="maxLoginAttempt" value="<?= Yii::app()->getConfig('maxLoginAttempt') ?>" />
+                        <input class="form-control" type="number" min="1" step="1" pattern="^\d*$" name="maxLoginAttempt" placeholder="<?= gT("Disabled") ?>"
+                            value="<?= App()->getConfig('maxLoginAttempt') !== "" ?? intval(App()->getConfig('maxLoginAttempt')) ?>"
+                        />
+                        <div class="form-text"><?= gT("Set an empty value disable brute force protection. Number of attemps are deleted each time.") ?></div>
```

Comment:
Please write this sentence like this:
"Set an empty value to disable brute force protection. Number of attempts are deleted each time."

Reply:
Oups … there are an issue in this sentence see https://github.com/LimeSurvey/LimeSurvey/pull/3617/files#diff-9277e62f270b5ccca6372e2d60aaaf7b1e48aded7cacd6536477f62e985186e1R141

It must be _"Set an empty value disable brute force protection. Number of attempts are never checked."_  both disable the system but not on the same way.

Reply:
_Set an empty value disable brute force protection. Number of attempts are never checked._

Reply:
Fixed by _Set an empty value to disable brute force protection.  Number of attempts are never checked._